jekyll/_cci2/configuration-reference.adoc

```diff
@@ -2713,10 +2718,10 @@ NOTE: Workflows will ignore job-level branching. If you use job-level branching
 | `filters`
 | N
 | Map
-| A map defining rules for execution on specific branches
+| A map defining rules for execution on specific branches, tags, or based on an expression-based condition.
```

Comment:
`filters` is now either a map, or a string. If it's a string then it's an expression filter, otherwise it's the older `branches`/`tags` structure.